airbyte-integrations/connectors/source-gitlab/source_gitlab/streams.py

```diff
@@ -403,7 +403,7 @@ class Users(GitlabChildStream):
 
 
 class Epics(GitlabChildStream):
-    primary_key = "iid"
```

Comment:
Are you sure this should be `id`? [Gitlab API](https://docs.gitlab.com/ee/api/epics.html) operates IIDs in the URLs.